examples/openSUSE/microos.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: microos
+  namespace: system-upgrade
+type: Opaque
+stringData:
+  upgrade.sh: |
+    #!/bin/sh
+    set -e
+    secrets=$(dirname $0)
+    transactional-update --continue cleanup dup
+    # For a delayed reboot use:
+    #   rebootmgrctl reboot
+    # or chain it with transactional-update commands:
+    #   transactional-update --continue cleanup dup reboot
+    [ -f /run/reboot-needed ] && rebootmgrctl reboot now
+---
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: microos
+  namespace: system-upgrade
+spec:
+  concurrency: 2
+  nodeSelector:
+    matchExpressions:
+      - {key: node-role.kubernetes.io/worker, operator: In, values: ["true"]}
```

Comment:
Are you sure about this? RKE2 and K3s do not use the 'worker' role so this will never match any nodes.

Reply:
I label it as such, but I used the same that is in https://github.com/rancher/system-upgrade-controller/blob/master/examples/suse/sles.yaml#L26.

Can something like this be better? https://github.com/rancher/system-upgrade-controller/blob/master/examples/ubuntu/bionic.yaml#L31

Reply:
I don't know, none of these labels will exist by default - I guess the examples assume people understand that they need to label their nodes to match the LabelSelector in the plan? I don't mean to make you fix all of this; I approved so you can merge if you want .

Reply:
Excellent. I do not have rights to merge it. Can you do that?

Thanks!